pipeline/alert.go

```diff
@@ -949,6 +949,7 @@ func (a *AlertaHandler) Services(service ...string) *AlertaHandler {
 //      enabled = true
 //      url = "http://sensu:3030"
 //      source = "Kapacitor"
+//      handlers = "sns"
```

Comment:
Good questions. I checked in some code that would allow for a list of handlers. I only needed one, but lists work as well.
I could understand a need for having different handlers per alert, but we only need a single global one at the moment.